selfie.c

```diff
@@ -3985,6 +4021,15 @@ void emitLeftShiftBy(int b) {
   emitRFormat(OP_SPECIAL, 0, 0, currentTemporary(), FCT_MFLO);
 }
 
+void emitLeftShiftByLterm(int b) {
```

Comment:
Generalize `emitLeftShiftBy(int b)` to emitLeftShiftBy(int reg, int b)` and call `emitLeftShiftBy(currentTemporary(), 2)` and `emitLeftShiftBy(previousTemporary(), 2)` accordingly.